googletest/include/gtest/gtest-printers.h

```diff
@@ -423,9 +423,9 @@ void DefaultPrintTo(IsNotContainer /* dummy */,
       // void*.  However, we cannot cast it to const void* directly,
       // even using reinterpret_cast, as earlier versions of gcc
       // (e.g. 3.4.5) cannot compile the cast when p is a function
```

Comment:
No tears shed on loss of GCC 3.4.5 here!

Let me check real quick if MSVC complains about that. C is kinda unsure about the whole von Neumann architecture thing and doesn't promise data and function pointers are the same size. My experience is GCC and Clang don't care but MSVC, of all compilers, sometimes gets grumpy about this. :-(